kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyConfigSecret.java

```diff
@@ -55,6 +59,14 @@
 public class ProxyConfigSecret
```

Comment:
With this change we no longer need to pass the generated proxy config to the container in a Secret, because all the sensitive info can be put in secrets. I intend to change this class to use a ConfigMap in a future PR.

Reply:
Created #1890 to track it